File: inflate/tests/langs/perl/box.pl
use strict;
use warnings;

package Box {
    sub helper {
        my ($value) = @_;
        return $value + 1;
    }

    sub greet {
        my ($value) = @_;
        return helper($value) * 2;
    }

    sub runner {
        return greet(3);
    }
}

1;

File: inflate/tests/langs/typescript/box.ts
export class Box {
    private size = 1;

    helper(value: number): number {
        return value + this.size;
    }

    greet(value: number): number {
        return this.helper(value) * 2;
    }

    runner(): number {
        return this.greet(3);
    }
}
